arviz/plots/backends/bokeh/distplot.py

```diff
@@ -119,8 +128,87 @@ def _histplot_bokeh_op(values, values2, rotated, ax, hist_kwargs):
     if hist_kwargs.pop("cumulative", False):
         hist = np.cumsum(hist)
         hist /= hist[-1]
-    if rotated:
-        ax.quad(top=edges[:-1], bottom=edges[1:], left=0, right=hist, **hist_kwargs)
+
+    if is_circular:
+
+        if is_circular == "degrees":
+            edges = np.deg2rad(edges)
+
+        start_angle = edges
+        outer_radius = hist
+
+        ax.annular_wedge(
+            x=0,
+            y=0,
+            inner_radius=0,
+            outer_radius=outer_radius,
+            start_angle=start_angle,
+            end_angle=start_angle + 0.8,
+            **hist_kwargs,
+        )
+
+        ticks = np.linspace(-np.pi, np.pi, 9)
```

Comment:
This could be

`ticks = np.linspace(-np.pi, np.pi, len(labels), endpoint=False)`